qemu/tests/block_commit_stress.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from virttest import utils_misc
+from virttest import utils_test
+
+from qemu.tests import blk_commit
+
+
+class BlockCommitStress(blk_commit.BlockCommit):
+
+    def load_stress(self):
+        """
+        load IO/CPU/Memoery stress in guest
+        """
+        logging.info("launch stress app in guest")
+        args = (self.test, self.params, self.env, self.params["stress_test"])
+        bg_test = utils_test.BackgroundTest(utils_test.run_virt_sub_test, args)
+        bg_test.start()
+        if not utils_misc.wait_for(bg_test.is_alive, first=10, step=3, timeout=100):
+            self.test.fail("background test start failed")
+        if not utils_misc.wait_for(self.app_running, timeout=360, step=5):
+            self.test.fail("stress app isn't running")
+
+    def unload_stress(self):
+        """
+        stop stress app
+        """
+        def _unload_stress():
+            session = self.get_session()
+            cmd = self.params.get("stop_cmd")
+            session.sendline(cmd)
+            session.close()
+            return self.app_running()
+
+        logging.info("stop stress app in guest")
+        stopped = utils_misc.wait_for(_unload_stress, first=2.0,
+                                      text="wait stress app quit", step=1.0,
+                                      timeout=self.params["wait_timeout"])
+        if not stopped:
+            logging.warn("stress app is still running")
+
+    def app_running(self):
+        """
+        check stress app really run in background
+        """
+        session = self.get_session()
+        cmd = self.params.get("check_cmd")
+        status = session.cmd_status(cmd, timeout=120)
+        session.close()
+        return status == 0
+
+    def verify_backingfile(self):
+        """
+        check no backingfile found after commit job done via qemu-img info;
+        """
+        logging.info("Check image backing-file")
+        exp_img_file = self.params.get("expected_image_file")
```

Comment:
If this is a required param, please reference it by `[]`, otherwise, should give the default value.